app/cli/internal/action/attestation_status.go

```diff
@@ -99,26 +99,14 @@ func (action *AttestationStatus) Run(ctx context.Context, attestationID string)
 		Annotations:   pbAnnotationsToAction(c.CraftingState.InputSchema.GetAnnotations()),
 	}
 
-	// Materials
-	for _, m := range c.CraftingState.InputSchema.Materials {
-		materialResult := &AttestationStatusResultMaterial{
-			Material: &Material{
-				Name: m.Name, Type: m.Type.String(),
-				Annotations: pbAnnotationsToAction(m.Annotations),
-			},
-			IsOutput: m.Output, Required: !m.Optional,
-		}
-
-		// If it has been added already we load the value
-		if cm, found := c.CraftingState.Attestation.Materials[m.Name]; found {
-			if err := setMaterialValue(cm, materialResult.Material); err != nil {
-				return nil, err
-			}
-			materialResult.Set = true
-			materialResult.Tag = cm.GetContainerImage().GetTag()
-		}
+	// Temporal map to keep track of visited materials
```

Comment:
I'd add a comment explaining why we need to do lookups on basically two collections.

We want to make sure we show all the expected materials, that's why we load them from the contract

but also want to show additional items that might not be in the contract once they are set.